src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -419,3 +419,32 @@ databaseChangeLog:
       changes:
         - tagDatabase:
             tag: version_0.1.0
+  - changeSet:
+      id: sneaky-view
+      author: ben.warfield@usds.dhs.gov
+      comment: This had better not make it directly into prod.
+      runOnChange: true
+      changes:
+        - sql:
+            sql: drop view if exists ${database.defaultSchemaName}.sneaky_view
+        - createView:
+            viewName: sneaky_view
```

Comment:
I intentionally used "Sneaky" to force myself to have a discussion about the name later, so yes, 💯

Ironically, `CaseSummary` is not a bad name for it, but that's taken already. Maybe `LatestCaseData`? Or maybe CaseSummary is due for getting a sincere kick in the teeth anyway....

Reply:
This brings up a good point. Arn't we building `CaseSummary` with this query. I mean the difference between the `sneaky_view` and `CaseSummary` is that `CaseSummary` joins with `Attachments` and `Users`. 

Unless I am missing something it seems like it would be better to do that join in the database instead of the Java logic. And then we could get rid of the old CaseSummary?

Reply:
We could do it, with some hassle, in the persistence layer—this would probably result in an N+1 query situation, though. Doing it as part of the underlying database view, using a subselect that flattens its results into a JSON array, is also possible, but I'm sort of assuming you didn't mean that. 😁